CHANGELOG.md

```diff
@@ -77,6 +78,30 @@ title: Changelog
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 3.11.0
+
+### Plugins
+
+- allow configuring keepalive_timeout [#11611](https://github.com/apache/apisix/pull/11611)
+- jwt-auth plugin no longer requires a private_key to be uploaded. [#11597](https://github.com/apache/apisix/pull/11597)
```

Comment:
Should move to under a subtitle `Change` like https://github.com/apache/apisix/blob/master/CHANGELOG.md#change-2